charts/stellar-network/values.yaml

```diff
@@ -1,25 +1,29 @@
 global:
   # namespace: default
-  networkPassphrase: "Test SDF Network ; September 2015"
-  horizonUrl: "https://horizon-testnet.stellar.org"
+  networkPassphrase: "Reynholm Industries Network ; January 2023"
+  horizonUrl: "http://reynholm-industries-horizon-ingest:80/"
```

Comment:
This should be overriden in the values-reynholmindustries.yaml file not in the horizon chart's values.